File: src/main/java/night/kotlin/seminar/javaexample/PowExample.java
package night.kotlin.seminar.javaexample;

import java.util.List;
import java.util.stream.StreamSupport;

public class PowExample {
	public static void main(String[] args) {
		StreamSupport
			.stream(List.of(1, 2, 3, 4, 5, 6, 7, 8, 9).spliterator(), false)
			.map(it -> it * it)
			.forEach(System.out::println);

	}
}

File: src/main/kotlin/night/kotlin/seminar/kotlin/PowExample.kt
package night.kotlin.seminar.kotlin

import java.util.stream.Stream
import kotlin.streams.toList

fun main() {
    val powList = Stream.of(1, 2, 3, 4, 5, 6, 7, 8, 9)
            .map { it * it }
            .toList()

    for (pow in powList) {
        println(pow)
    }
}

